src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarTownEventListener.java

```diff
@@ -533,4 +536,12 @@ public void on(TownMapColourCalculationEvent event) {
 			event.setMapColorHexCode(mapColorHexCode);
 		}
 	}
+
+	private static boolean battleSessionIsActive(Siege siege) {

```

Comment:
:construction: 
- This method gives when the "battle" is active rather than the "battle session"
- Thus pls rename to "battleIsActive()"